.github/workflows/awie.yml

```
@@ -172,7 +172,11 @@
       contains(fromJson('["testing", "unstable"]'), needs.get-environment.outputs.stability) &&
       needs.changes.outputs.trigger_delivery == 'true' &&
       github.repository == 'centreon/centreon'
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 175: could not find expected ':'

(syntax-check)